drivers/media/i2c/adv7180.c

```diff
@@ -1469,7 +1550,8 @@ static int adv7180_resume(struct device *dev)
 	struct v4l2_subdev *sd = i2c_get_clientdata(client);
 	struct adv7180_state *state = to_state(sd);
 	int ret;
-
+	
+	v4l_info(client,"Resuming ADV device");
```

Comment:
Nor add debug messages here.

Reply:
Sorry these three were for debug I forgot to revert them. I'm happy for this to not be pulled as I've clearly implemented it incorrectly.

Reply:
Thanks for trying - it's the way to learn.
It's only though years of watching upstream and working with the APIs that I know the "proper way" to implement these things - they aren't always obvious. Hacks are reserved for where there isn't a "proper way" (eg ```dbg_input``` and I'm now looking to fix that properly).

Where drivers have the potential to be upstreamed, or are upstream drivers in the first place, we do try to keep to the coding standards and play by upstream's rules. It's a bit of a slog to start with, and you'll be swearing at ```checkpatch``` for a while, but it allows others to make use of your work and try upstreaming it without ambiguity.